File: b_XcF1A1nqCfu-1775104848425/scripts/install-ai-deps.mjs
import { execSync } from 'child_process'
import path from 'path'

const projectDir = '/vercel/share/v0-project'

console.log('[v0] Installing AI SDK packages...')
try {
  const result = execSync(
    'pnpm add @ai-sdk/groq@^1.2.9 ai@^6.0.0',
    { cwd: projectDir, encoding: 'utf8', stdio: 'pipe' }
  )
  console.log('[v0] Install output:', result)
  console.log('[v0] Successfully installed @ai-sdk/groq and ai@6')
} catch (err) {
  console.error('[v0] Install failed:', err.message)
  console.error('[v0] stderr:', err.stderr)
}

File: scripts/install_ai_deps.py
import subprocess, sys, os

project_dir = '/vercel/share/v0-project'

print('[v0] Installing AI SDK packages...')
result = subprocess.run(
    ['pnpm', 'add', '@ai-sdk/groq@^1.2.9', 'ai@^6.0.0'],
    cwd=project_dir,
    capture_output=True,
    text=True
)

print('[v0] stdout:', result.stdout)
print('[v0] stderr:', result.stderr)
print('[v0] return code:', result.returncode)

if result.returncode == 0:
    print('[v0] SUCCESS: packages installed')
else:
    print('[v0] FAILED: check stderr above')
    sys.exit(1)
